app/javascript/components/patient/laboratory/LaboratoryModal.js

```diff
@@ -120,6 +130,30 @@ class LaboratoryModal extends React.Component {
                 </Form.Control>
               </Form.Group>
             </Row>
+            {this.props.editMode &&
+              this.props.isolation &&
+              this.props.onlyPosLab &&
+              (this.state.result !== 'positive' || this.state.specimen_collection == null) && (
+                <div>
+                  <p className="mt-2 mb-3">
+                    Since this record does not have a Symptom Onset Date, updating this lab from a positive result or clearing the specimen collection date may
+                    result in the record not ever being eligible to appear on the Records Requiring Review line list. Please undo these changes or enter a
+                    symptom onset date:
+                  </p>
+                  <Form.Label className="nav-input-label">SYMPTOM ONSET</Form.Label>
```

Comment:
is this label necessary?  I think it looks strange and the paragraph before explains the date input

Reply:
https://mitrecorp.slack.com/archives/C01A6P94R9B/p1623861467059600?thread_ts=1623785133.027100&cid=C01A6P94R9B